WORKSPACE.bazel

```diff
@@ -26,9 +26,9 @@ http_archive(
     ],
 )
 
-LLVM_COMMIT = "f81f32adc9a8e99a77dd4c5f5f83c5595b989a71"
+LLVM_COMMIT = "afb73f7a913ec8e7e8704afe18784571f320ebf6"
```

Comment:
Since this PR is still open, can we integrate a more recent commit, e.g. https://github.com/tensorflow/mlir-hlo/commit/361cf495bd110d374968c2b111ad89e085033072?